### PR TITLE
March 2024 recap

### PR DESCRIPTION
**Pull request by lenart**

### CODE REVIEW

File: _posts/2024-03-25-march-meetup.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title: March Ruby Meetup
+author: lenartr
+bg_photo: /assets/img/posts/2024-03-25-meetup.jpg
+---
+
+In March another Ruby Meetup brought together familiar faces and fresh ideas in the vibrant world of Ruby development. As always the atmosphere inside was relaxed and buzzing with excitement.
+
+One of the highlights of the evening was Radan Skoric's talk, "Should I type or should I not?" Radan delved into the realm of typing in Ruby, exploring various approaches and their pros and cons. From Rbs to Sorbet, he presented a comprehensive overview of how types can be introduced into Rails projects, sparking lively discussions among attendees.
```
Comment (miharekar) [inline _posts/2024-03-25-march-meetup.md:10]: I'd disagree about it being a highlight but oh well 😂

On a serious note, I would link to his blog though: https://radanskoric.com/

Comment (MrChriss) [reply _posts/2024-03-25-march-meetup.md:10]: what @miharekar said :heavy_check_mark:

**Review by miharekar — APPROVED** [review]
:shipit:

**Comment by lenart** [comment]
> Only those with [write access](https://docs.github.com/articles/what-are-the-different-access-permissions) to this repository can merge pull requests.

😉

**Pull request merged**